rosco-manifests/rosco-manifests.gradle

```diff
@@ -14,11 +14,14 @@ dependencies {
   implementation "org.yaml:snakeyaml"
 
   implementation "com.squareup.retrofit:retrofit"
+  implementation "com.squareup.retrofit2:retrofit"
   testImplementation "org.assertj:assertj-core"
   testImplementation "org.junit.jupiter:junit-jupiter-api"
   testImplementation "org.junit.jupiter:junit-jupiter-params"
   testImplementation "org.junit.platform:junit-platform-runner"
   testImplementation "org.junit.vintage:junit-vintage-engine"
   testImplementation "org.mockito:mockito-core"
+  testImplementation "org.mockito:mockito-inline"
```

Comment:
There's a convention of using [a mockito-extensions file](https://github.com/spinnaker/rosco/blob/368f8e59ddf81eb1e8b72592fb2df209238a7f14/rosco-web/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker) instead of this dependency.  Mind doing it this way instead?